_account-security/security/encryption/ssh-for-amazon-web-services.md

```diff
@@ -0,0 +1,231 @@
+             - **Source**: This should default to **Custom**. In the field next to the Source drop-down menu, paste one of the following IP addresses:
+
+                {% for ip-address in ip-addresses %}
+                - {{ ip-address.ip }}
+                {% endfor %}
+
+          4. Click the **Add Rule** button to add another rule.
+          5. Repeat steps 3 and 4 until all of Stitch's IP addresses have been added to the Security Group.
+
+             Here's what the Security Group rules should look like:
+
+             ![Configuring the EC2 Instance Security Group]({{ site.baseurl }}/images/shared/ssh/amazon-ec2-security-group.png)
+          6. When finished, click the **Review and Launch** button in the lower right corner of the page.
+
+      - title: "Review and launch the SSH server"
+        anchor: "review-launch-ssh-server"
+        content: |
+          The last step is to review the settings for the SSH server and launch it.
+
+          Review the instance's settings, paying particular attention to the fields highlighted in the image below:
+
+          ![Reviewing the EC2 Instance Details]({{ site.baseurl }}/images/shared/ssh/amazon-ec2-instance-review.png)
+
+          - The **Security Groups** section should list either a new Security Group for Stitch OR an existing group that contains group rules for Stitch's IP addresses. If it doesn't, [refer to Step 2.2](#ec2-instance-security-group) for instructions.
+          - In the **Instance Details** section:
+             - **Network**: This field should contain the ID of the same VPC that the database is in.
+             - **Subnet**: This field should contain the ID of a public subnet.
+             - **Assign Public IP & Assign IPv6 IP:** We strongly recommend using a [public subnet with the instance](#configure-ec2-instance) and auto-assigning a public IP address. This will ensure that Stitch can access the instance.
+
+          After you're reviewed the instance's settings, click the **Launch** button in the lower right corner to launch the instance.
+
+          **Note**: It may take a few minutes for the instance creation process to complete. The status in the VPC Dashboard page will change to `Available` when the instance is ready.
+
+  - title: "Enable the SSH server to access the database"
+    anchor: "enable-ssh-server-access"
+    content: |
+      After the EC2 instance has finished initializing, you can move onto configuring the access rules for database. In this section, you'll create a VPC Security Group that will forward traffic from the SSH server (EC2 instance) to the database in the private subnet.
+    
+    substeps:
+      - title: "Retrieve the VPC's IPv4 CIDR"
+        anchor: "retrieve-vpc-ip"
+        content: |
+          In this step, you'll retrieve the SSH server's IP address, or IPv4 CIDR. This value will be followed by a slash and a number between 0 and 32. For example: `10.0.0.0/16`
```

Comment:
these instructions actually have the user whitelist the CIDR for the *entire VPC*. This will include the SSH tunnel instance, so it will work, but it might be more permissive than they want

Reply:
1. Get Private IP from the bastion
2. Add `/32` to the IP